### ISSUE TITLE
[Enhancement]: Support MySQL binlog format

### ISSUE THREAD

**Issue by wenchaomeng** [oceanbase/oceanbase#1648]
MySQL binlog is widely used in a lot of tools.  Massive work is required to adapt to oblogproxy.

**Comment by namewxt1220** [comment]
Yes, you are right, mysql binlog format will be released and supported in the near future, the core component of which is oblogproxy.

**Comment by finovy** [comment]
Hello, may I inquire if there is a tentative release date for the oblogproxy tool?

**Comment by namewxt1220** [comment]
Please pay attention to the [oblogproxy](https://github.com/oceanbase/oblogproxy) repository and the latest [release](https://github.com/oceanbase/oblogproxy/releases) information.
